### PR TITLE
Enable specific bots to run ClemBot commands

### PR DESCRIPTION
**Pull request by jkriste**

### CODE REVIEW

File: ClemBot.Bot/bot/clem_bot.py [hunk 1]
```diff
@@ -201,6 +201,10 @@ async def claims_check(self, ctx: ext.ClemBotCtx) -> bool:
         command = ctx.command
         author = ctx.author
 
+        # For bots that are allowed to run commands
+        if ctx.author.bot:
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/clem_bot.py:205]: Why do we need this?

File: ClemBot.Bot/bot/services/message_handling_service.py [hunk 2]
```diff
@@ -90,6 +91,11 @@ async def on_guild_message_received(self, message: discord.Message) -> None:
         # Enabled embed message links
         await self.handle_message_links(message)
 
+        # Check if author is bot and is in our allowed bot input ids
+        if message.author.bot and message.author.id in bot_secrets.secrets.allow_bot_input_ids:
+            ctx = await self.bot.get_context(message)
+            await self.bot.invoke(ctx)
+
         # Primary entry point for handling commands
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/services/message_handling_service.py:99]: This is invoking a command twice

Comment (jkriste) [reply ClemBot.Bot/bot/services/message_handling_service.py:99]: The user either is or isn't a bot, and process_command will return if the user is a bot, so it'll only run once.

File: ClemBot.Bot/bot/bot_secrets.py [hunk 3]
```diff
@@ -244,6 +258,7 @@ def load_production_secrets(self) -> None:
         self.api_key = os.environ.get("API_KEY")  # type: ignore
         self.site_url = os.environ.get("SITE_URL")  # type: ignore
         self.docs_url = os.environ.get("DOCS_URL")  # type: ignore
+        self.allow_bot_input_ids = os.environ.get("ALLOW_BOT_INPUT_IDS")  # type: ignore
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/bot_secrets.py:261]: This wont work, it has to insert a list of ints. this inserts a string, see line 248 for example

**Review by Jay-Madden — APPROVED** [review]
lgtm

**Pull request merged**